cmd/utils/flags.go

```diff
@@ -130,6 +130,10 @@ var (
 		Usage: "Explicitly set network id (integer)(For testnets: use --garden)",
 		Value: ethconfig.Defaults.NetworkId,
 	}
+	SlicesRunningFlag = cli.StringFlag{
+		Name:  "slices",
+		Usage: "All the slices that are available for data fetch request from a peer",
```

Comment:
This isn't just about data fetch requests, right? This is literally specifying which slices to run. The note should say that.